jsonnet/main.jsonnet

```diff
@@ -64,6 +67,7 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
                  promLabelProxy: 'v0.1.0',
                  kubeRbacProxy: 'v0.4.1',
                  prometheusAdapter: 'v0.4.1',
+                 thanos: 'latest',
```

Comment:
We should probably be more specific about version used (v0.6.0 ?)

Reply:
this gets overriden by image-references, but for good hygiene i agree, let's pin the version here.

Reply:
oh, wait, we are referencing https://quay.io/repository/openshift/origin-thanos?tag=latest&tab=tags, there's no upstream versions here, I'll rename the var to `openshiftThanos` to avoid confusion.